ec/acc/deployment_test.go

```diff
@@ -22,65 +22,10 @@ import (
 	"io/ioutil"
 	"testing"
 
-	"github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi"
-	"github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/deputil"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-const (
-	deploymentVersion          = "7.8.1"
-	deploymentVersionAppsearch = "7.6.2"
-	region                     = "us-east-1"
-)
-
-func TestAccDeployment_basic(t *testing.T) {
```

Comment:
If this test was extracted into a separate file, should the other tests be extracted too?

Reply:
yeah, but it's out of the scope of this PR